HidSharp/Platform/MacOS/NativeMethods.cs

```diff
@@ -495,6 +560,9 @@ public static byte[] IORegistryEntryGetCFProperty_Data(int entry, IntPtr dataKey
         [DllImport(IOKit, EntryPoint = "IOServiceMatching")] // name = IOHIDDevice
         public static extern IntPtr IOServiceMatching([MarshalAs(UnmanagedType.LPStr)] string name);
 
+        [DllImport(IOKit, EntryPoint = "QueryInterface")]
+        public static extern int QueryInterface(IntPtr a, IntPtr b, ref IntPtr c);
```

Comment:
```
Exception has occurred: CLR/System.EntryPointNotFoundException
Exception thrown: 'System.EntryPointNotFoundException' in HidSharpCore.dll: 'Unable to find an entry point named 'QueryInterface' in shared library '/System/Library/Frameworks/IOKit.framework/IOKit'.'
   at HidSharp.Platform.MacOS.NativeMethods.QueryInterface(IntPtr a, IntPtr b, IntPtr& c)
   at HidSharp.Platform.MacOS.MacHidDevice.GetDeviceString(Int32 index) in /Users/infinity/Documents/Repos/HidSharpCore/HidSharp/Platform/MacOS/MacHidDevice.cs:line 207
```

Reply:
Maybe QueryInterface is reference to [Marshal.QueryInterface()](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.marshal.queryinterface?view=netcore-3.1)?

Reply:
New changes use Marshal now.